zipline/gens/tradesimulation.py

```diff
@@ -180,6 +184,10 @@ def handle_benchmark(date):
             stack.enter_context(self.processor)
             stack.enter_context(ZiplineAPI(self.algo))
             for dt, action in self.clock:
+
+                if algo.data_frequency == 'minute':
```

Comment:
can you move this to the `DAY_END` handler?  worried about the perf impact of checking every minute.  if we eventually have intra-day cancellation policies, we can revisit this.